_posts/2016-12-06-recurse-center-week-10.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Week[9] @ The Recurse Center 🔎"
+comments: true
+categories: tech recurse
+---
+
+### Overall
+
+I'm currently trying to distill through what is essential, and focus on that. Stripping away the excess has left more energy for the important pursuits.
```

Comment:
How does this sentence sound instead: _I'm currently trying to distill what is essential, and focusing on that_?